### PR TITLE
Add CSM - Central Saint Martins - Update facilities.json

### PR DESCRIPTION
**Pull request by japierce1028**
Opt IN - only City/country for address.

### CODE REVIEW

File: src/main/data/facilities.json [hunk 1]
```diff
@@ -1516,6 +1516,16 @@
       "FAC"
     ]
   },
+  {
+    "code": "CSM",
+    "contact": {
+      "address": "London, United Kingdom",
```
Comment (palemieux) [inline src/main/data/facilities.json:1522]:
```suggestion
```

Let's remove the incomplete address.

**Review by palemieux — APPROVED** [review]

File: src/main/data/facilities.json (hunk 1 above)
Comment (japierce1028) [reply src/main/data/facilities.json:1522]: I think it should remain. I think it gives the region of interest to folks. Granted it's not the mailing address I think there is value to City/State.

Comment (palemieux) [reply src/main/data/facilities.json:1522]: It breaks automation that assumes that a complete mailing address is present. The submitter should submit a complete address.

Comment (japierce1028) [reply src/main/data/facilities.json]: I have added a full address...

**Review by palemieux — APPROVED** [review]

**Pull request merged**